src/renderer/rendered_window.rs

```diff
@@ -285,25 +285,28 @@ impl RenderedWindow {
         let pixel_region = self.pixel_region(font_dimensions);
         let transparent_floating = self.anchor_info.is_some() && has_transparency;
 
-        if self.anchor_info.is_some() {
+        if self.anchor_info.is_some() && settings.floating_shadow {
             root_canvas.save();
             let shadow_path = Path::rect(pixel_region, None);
             // We clip using the Difference op to make sure that the shadow isn't rendered inside
             // the window itself.
             root_canvas.clip_path(&shadow_path, Some(ClipOp::Difference), None);
+            // The light angle is specified in degrees from the vertical, so we first convert them
+            // to radians and then use sin/cos to get the y and z components of the light
+            let light_angle_radians = settings.light_angle_degrees * std::f32::consts::PI / 180.;
```

Comment:
Could use `f32::to_radians`.

```suggestion
            let light_angle_radians = settings.light_angle_degrees.to_radians();
```